eng/scripts/build.ps1

```diff
@@ -16,19 +24,33 @@ if (![string]::IsNullOrWhiteSpace($filter)) {
     $keys = $keys.Where( { $_ -match $filter }) 
 }
 
-$keys | ForEach-Object { $sdks[$_] } | ForEach-Object -Parallel {
+$keys | ForEach-Object { $sdks[$_] } | ForEach-Object {
     Push-Location $_.path
 
-    if (!$skipBuild) {
+    if ($_.clean) {
+        Write-Host "##[command]Executing go clean -v ./... in " $_.path
+        go clean -v ./...
+    }
+
+    if ($_.generate) {
+        Write-Host "##[command]Executing autorest.go in " $_.path
+        $autorestPath = $_.path + "\autorest.md"
+        $autorestVersion = "@autorest/go@4.0.0-preview.23"
+        $outputFolder = $_.path
+        $root = $_.root
+        autorest --use=$autorestVersion --go --track2 --go-sdk-folder=$root --output-folder=$outputFolder --file-prefix="zz_generated_" --clear-output-folder=false $autorestPath
+    }
+    if (!$_.skipBuild) {
         Write-Host "##[command]Executing go build -v ./... in " $_.path
-        go build -v ./...
+        go build -x -v ./...
+        Write-Host "##[command]Build Complete!"
+
     }
-    if ($vet) {
+    if ($_.vet) {
         Write-Host "##[command]Executing go vet ./... in " $_.path
         go vet ./...
     }
-    if ($generate) {
-        Write-Host "##[command]Executing autorest.go in " $_.path
-        # TODO
-    }
-} -ThrottleLimit $parallel
+    Pop-Location
+}
+
+Set-Location $startingDirectory
```

Comment:
Similar to my above comment re: fail fast, is the script intended to be run with ErrorAction Continue? If not, it will dump you out into a sub-directory on failure (assuming it's ever intended to be ran locally).

Reply:
It will dump you in a subdirectory on failure, and this is bad.